virtcontainers/sandbox.go

```diff
@@ -1062,6 +1063,26 @@ func (s *Sandbox) stopVM() error {
 		s.Logger().Info("Not stopping VM")
 		return nil
 	}
+	pid := s.hypervisor.pid()
+
+	defer func() {
+		// if use PhysicalEndpoint, wait for hypervisor exited
+		// https://work.aone.alibaba-inc.com/issue/20349397
+		exited := false
+		for i := 0; i < 50; i++ {
+			if running, _ := isShimRunning(pid); !running {
```

Comment:
here we are relying that the `shim` dies when the hypervisor dies, this is prone to bugs